conmon-rs/common/proto/conmon.capnp

```diff
@@ -24,6 +24,7 @@ interface Conmon {
 
     struct CreateContainerResponse {
         containerPid @0 :UInt32;
+        attachSocketPath @1 :Text;
```

Comment:
alternative to always returning as create container response: we create an AttachSocket request that a client could use to create an attach socket for the given create/exec session. I am open to either, what are your thoughts @saschagrunert

Reply:
Yeah moved towards that :+1: